features/login.feature

```diff
@@ -15,3 +15,10 @@ Feature: Logging into system
     Then he should see "You have one more attempt before your account is locked"
     When he attempts to log-in 3 times with incorrect password
     Then he should see "Your account is locked"
+
+  Scenario: Admin disables account
+    Given the user 'foouser@example.com' has an account
+    When his account has been disabled
+    And he attempts to log-in with correct details
+    Then he should see "Access to your account has been suspended, please refer to your supervisor"
+
```

Comment:
Thanks for the feedback on Cucumber. Believe it or not I'm not 100% sure about it either ^_^. I am however a test-first zealot and when I'm adding functionality via 3rd party gem this seemed perfect way to at least test something.

In all my years in ruby I've been lucky enough to work in some pretty diverse teams. I spent the first 4 years 100% anti-cucumber. Last year I got to work in a place where the QA team was very technical - they were all 'developers in test'. They convinced me the  [Testing Triangle](https://www.mountaingoatsoftware.com/blog/the-forgotten-layer-of-the-test-automation-pyramid) is the way to work effectively.

My preferred workflow is to write a simple feature test that touches all components. I'm guilty of not writing enough 'service' tests (RSpec features in this case) but I do make up with good unit tests I hope. I think the main objection to cucumber is the work required to set up tests but now I'm more experienced I don't think this has to be the case.